projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx

```diff
@@ -82,7 +81,7 @@ export default function VideoPressEdit( {
 	const { createErrorNotice } = useDispatch( noticesStore );
 
 	const videoPressUrl = getVideoPressUrl( guid, {
-		autoplay,
+		autoplay: false,
```

Comment:
Nitpick: It would be great to add a comment here to briefly explain why we enforce disabling `autoplay`, WDYT?

Reply:
fixed in https://github.com/Automattic/jetpack/pull/29823/commits/03e6e67bdf77b12d8442125a9fc9bd6209a5f2ab